pkgs/by-name/ud/udev-block-notify/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  multimarkdown,
+  libnotify,
+  udev,
+  systemdLibs,
+  glib,
+  pkg-config,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "udev-block-notify";
+  version = "0.7.11";
+
+  src = fetchFromGitHub {
+    owner = "eworm-de";
+    repo = "udev-block-notify";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-A0uhfb2mEAAJgxRkv+MWTk/9oFiz3r7deAlu1Kpk+CI=";
+  };
+
+  nativeBuildInputs = [
+    multimarkdown
+    pkg-config
+  ];
+  buildInputs = [
+    libnotify
+    udev
+    systemdLibs
+    glib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m0755 udev-block-notify $out/bin/udev-block-notify
+    install -D -m0644 systemd/udev-block-notify.service $out/lib/systemd/user/udev-block-notify.service
+
+    substituteInPlace systemd/udev-block-notify.service \
```

Comment:
Wait, you first install it, then replace the source file???

Reply:
I missed that, sorry